ref: document `dvc queue` (#3715)
iterative/dvc.org: content/docs/command-reference/queue/remove.md

## queue remove

Remove queued and completed tasks from the
[DVC Experiment](/doc/user-guide/experiment-management/experiments-overview)
task queue.

## Synopsis

```usage
usage: dvc queue remove [-h] [-q | -v]
                        [--all] [--queued] [--success] [--failed]
                        [<task> ...]

positional arguments:
  <task>         Tasks in queue to remove.
```

## Description

Removes the specified queued or completed experiment tasks from the queue. For
completed tasks, this will also remove any associated output logs.

> ⚠️ Note that for successfully completed tasks, this command is not the same as

> > `dvc exp remove`. This command not remove any Git or DVC data associated with

Reviewer: ```suggestion
> `dvc exp remove`. `dvc queue remove` does not remove any Git or DVC data associated with
```

> associated with a successful DVC experiment. It only removes the task queue
> entry and any associated output logs for that task.

## Options

- `--all` - remove all (queued and completed) experiment tasks from the queue.

- `--queued` - remove all queued experiment tasks from the queue.

- `--success` - remove all successfully completed tasks (and associated output

>   logs) from the queue.

Reviewer 2: - [ ] Probably too late to discuss now but just adding a note for a possible design follow-up:
  
  `queue remove --queued` sounds redundant. Maybe `--unprocessed/waiting` or something like that?  
  Also, `queue remove --success` -> `--successful`, I think.

Reviewer 2: This one was just an idea for you @dberenbaum, no follow-up needed in https://github.com/iterative/dvc.org/pull/3894.

Reviewer: It's a good point but not a high priority for me. Feel free to open an issue and maybe we can change it or make `--queued` a hidden alias when we have time.
